app/renderer/css/main.css

```diff
@@ -82,6 +82,59 @@ body {
     flex-direction: column;
 }
 
+
+/* DND Menu */
+
+/* stylelint-disable-next-line */
+@keyframes fading{0%{opacity:0}20%{opacity:1}80%{opacity:1}100%{opacity:0}}
```

Comment:
@manavmehta what happened to linting here?

Reply:
@akashnimare 
Toast fading out. I think I borrowed it from some module.
You want me to spread it ?

Reply:
No no, just run the linting that's it.

Reply:
There will be linting errors if stylelint is not disabled for that line. It requires spreading. With the exception, it gives no errors. Checked

Reply:
@manavmehta I think it should be something like this - 
```css
@keyframes fading {
    0% {
        opacity: 0
    }
    20% {
        opacity: 1
    }
    80% {
        opacity: 1
    }
    100% {
        opacity: 0
    }
}
```

Reply:
I had committed the changes, just waiting for either of you to suggest either to go for these or not. Pushed now. Thanks